iree/test/e2e/models/BUILD

```diff
@@ -67,6 +70,15 @@ iree_check_single_backend_test_suite(
     target_backend = "vulkan-spirv",
 )
 
+iree_check_single_backend_test_suite(
+    name = "check_linalg_on_tensors_vulkan-spirv_vulkan",
+    srcs = [
+        "mobilenetv2_fake_weights.mlir",
+    ],
+    driver = "vulkan",
+    target_backend = "vulkan-spirv",
```

Comment:
I think you need compiler_flags since this is named as `linalg_on_tensors_vulkan-spirv_vullkan`.

```
    compiler_flags = [
        "-iree-flow-dispatch-linalg-on-tensors",
        "-iree-codegen-spirv-experimental-linalg-on-tensors",
    ],
```

Reply:
And maybe add vectorization flag to it.

Reply:
Ah, I thought I added them.. Weird..

Reply:
Thanks for pointing out!